__tests__/app.test.js

```diff
@@ -155,7 +151,106 @@ describe('POST /api/articles/:article_id/comments', () => {
         .send(comment)
         .expect(201)
         .then(({body})=> {
-            const postedComment = body[0].body
+            const comment = body.comment;
+            const postedComment = comment[0].body
```

Comment:
Same as above here for sending a single comment back in an array. Send it back as an object something like {comment: comment[0]}